src/ape/contracts/base.py

```diff
@@ -13,6 +13,23 @@
     from ape.managers.networks import NetworkManager
 
 
+def _encode_address_args(*args):
+    """
+    Helper for encoding Ape AddressAPI to underlying address for ABI use
```

Comment:
Maybe we can make this more generic, like `_encode_input_args()`, just in case we have more of these type-mappings and stuff?

And `_encode_input_kwargs()`

Also, I think these doc strs a little distracting, maybe just comment like `# Convert higher level address types to 'str'`

Reply:
I went back and forth about that, but thought that if this was a one-off maybe it should be specific? If we wanted to do a more general solution it might be better to see what other issues need to be addressed and make a design change as well?